subprojects/parseq-restli-client/src/test/java/com/linkedin/restli/client/ParSeqRestClientIntegrationTest.java

```diff
@@ -160,6 +167,17 @@ protected void customizeEngine(EngineBuilder engineBuilder) {
     return _parseqClient.createTask(new GreetingsBuilders().get().id(id).build());
   }
 
+  protected Task<Response<Greeting>> greetingGetForDiffKeyType(String id) {
+    // This method is for the "testBatchingGetRequestsForDiffKeyType" test to create a request with different Key type
+    String _baseUriTemplate = "greetings";
+    HashMap<String, DynamicRecordMetadata> requestMetadataMap = new HashMap<>();
+    HashMap<String, DynamicRecordMetadata> responseMetadataMap = new HashMap<>();
+    HashMap<String, CompoundKey.TypeInfo> keyParts = new HashMap<>();
+    ResourceSpec _resourceSpec = new ResourceSpecImpl(EnumSet.of(ResourceMethod.GET, ResourceMethod.BATCH_GET, ResourceMethod.CREATE, ResourceMethod.BATCH_CREATE, ResourceMethod.PARTIAL_UPDATE, ResourceMethod.UPDATE, ResourceMethod.BATCH_UPDATE, ResourceMethod.DELETE, ResourceMethod.BATCH_PARTIAL_UPDATE, ResourceMethod.BATCH_DELETE, ResourceMethod.GET_ALL), requestMetadataMap, responseMetadataMap, String.class, (Class)null, (Class)null, Greeting.class, keyParts);
```

Comment:
This can be simplified as

```
new ResourceSpecImpl(EnumSet.allOf(ResourceMethod.class),
                  Collections.emptyMap(),
                  Collections.emptyMap(),
                  String.class,
                  null,
                  null,
                  Greeting.class,
                  Collections.emptyMap());
```